### PR TITLE
Bump RN version to 0.34.1

### PR DESCRIPTION
**Pull request by gilbox**

### CODE REVIEW

**Comment by gilbox** [comment]
@spikebrehm @lelandrichardson

**Comment by spikebrehm** [comment]
LGTM

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   ],
   "peerDependencies": {
     "react": ">=15.3.0",
-    "react-native": ">=0.32.0"
+    "react-native": ">=0.34.1"
```
Comment (spikebrehm) [inline package.json]: This means apps on 0.33 can't use 0.9.0 right?

Comment (gilbox) [inline package.json]: afaik you are correct.

**Comment by gilbox** [comment]
will do this in the future

**Pull request closed without merging**